<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageToAVIDialog</class>
 <widget class="QDialog" name="ImageToAVIDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Convert images to AVI</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../piaf.qrc">
    <normaloff>:/images/pixmaps/movie.png</normaloff>:/images/pixmaps/movie.png</iconset>
  </property>
  <property name="toolTip">
   <string>Encode a list of still image files into a MJPEG movie file</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Input image files</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="filesButton">
        <property name="toolTip">
         <string>Open a list of files</string>
        </property>
        <property name="text">
         <string>Files</string>
        </property>
        <property name="icon">
         <iconset resource="../../piaf.qrc">
          <normaloff>:/images/16x16/document-open.png</normaloff>:/images/16x16/document-open.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="dirButton">
        <property name="toolTip">
         <string>Open all files in directory</string>
        </property>
        <property name="text">
         <string>Directory</string>
        </property>
        <property name="icon">
         <iconset resource="../../piaf.qrc">
          <normaloff>:/images/16x16/document-open-folder.png</normaloff>:/images/16x16/document-open-folder.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="infoLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Status:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="statusIconLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="nbFilesLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="fpsSpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="singleStep">
         <number>5</number>
        </property>
        <property name="value">
         <number>25</number>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>fps</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Ouput movie file (MJPEG)</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="destButton">
        <property name="toolTip">
         <string>Destination movie file</string>
        </property>
        <property name="text">
         <string>Destination</string>
        </property>
        <property name="icon">
         <iconset resource="../../piaf.qrc">
          <normaloff>:/images/pixmaps/movie.png</normaloff>:/images/pixmaps/movie.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="destPathLineEdit">
        <property name="toolTip">
         <string>Destination file path</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="goButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Launch conversion</string>
        </property>
        <property name="text">
         <string>Go</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Conversion progression</string>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="toolTip">
      <string>Close dialog</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../piaf.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ImageToAVIDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ImageToAVIDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
